test/cypress/features/transaction.feature

```diff
@@ -0,0 +1,39 @@
+Feature: Transactions 
+
+  Background:
+    Given I login as testnet_guy on testnet
```

Comment:
I think this can be tested against dev net

Reply:
Done